### DISCUSSION TITLE
Inquiry Regarding Wireless Portable Ultrasound Compatibility with PlusToolkit, OpenIGT, and 3D Slicer

### DISCUSSION THREAD

**Discussion by enajwer** [PlusToolkit/PlusLib#1207] [Ideas]
Hello,

I am interested in using PlusToolkit, OpenIGT, and 3D Slicer with a wireless portable ultrasound device. So far, I have successfully used the libraries and software with a standard Telemed ultrasound machine. Could you kindly suggest which wireless ultrasound product would be easiest to adapt for these tools? Specifically, I am considering the Clarius wireless probe—do you think it would be a good solution?

Also, does PlusToolkit fully support wireless probes with WiFi communication, or would custom configuration be necessary for integration?

I would greatly appreciate your suggestions and guidance.

Thank you for your help.

Best regards,
Ewelina

**Comment by jonortega77** [comment]
I am currently using the Clarius C3 HD3 along with the [Clarius CAST API](https://github.com/clariusdev/cast) and works perfectly fine in windows 11 machine. It is perfectly compatible with PlusToolkit, OpenIGT and 3D Slicer.
